src/modules/test/recipes/create-asb-json.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
We need the copyright notice even here:

# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.

Reply:
done